roles/falcon_install/tasks/api.yml

```diff
@@ -84,7 +84,41 @@
       authorization: "Bearer {{ falcon_api_oauth2_token.json.access_token }}"
       Content-Type: application/json
   register: falcon_api_installer_list
-  no_log: yes
+  no_log: "{{ falcon_api_no_log }}"
+
+# Block for checking sensor/kernel compatibility
+- block:
+  - name: CrowdStrike Falcon | Build Sensor Update Kernels API Query
+    ansible.builtin.set_fact:
+      falcon_sensor_update_kernels_query: "{{ 'vendor:\"' + falcon_os_vendor + '\"+distro_version:\"' + ansible_distribution_version + '\"+release:\"' + ansible_kernel + '\"' }}"
+
+  - name: CrowdStrike Falcon | Get list of Supported Kernels
+    ansible.builtin.uri:
+      url: "https://{{ falcon_cloud }}/policy/combined/sensor-update-kernels/v1?filter={{ falcon_sensor_update_kernels_query | urlencode }}"
+      method: GET
+      return_content: true
+      headers:
+        authorization: "Bearer {{ falcon_api_oauth2_token.json.access_token }}"
+        Content-Type: application/json
+    register: falcon_sensor_update_kernels_list
+    no_log: "{{ falcon_api_no_log }}"
+
+  - name: CrowdStrike Falcon | Validate Kernel is Supported
```

Comment:
Seems as there are some ansible style errors with using `ignore_errors` instead of using `failed_when`

`ignore-errors Use failed_when and specify error conditions instead of using ignore_errors`

Reply:
solved by adding new variable to manage failures on these tasks